config/sass.js

```diff
@@ -2,9 +2,20 @@ module.exports = {
   options: {
     sourceMap: true,
   },
-  dist: {
+  dev: {
+    options: {
+      outputStyle: 'expanded'
+    },
     files: {
       'public/dist/main.css': 'public/scss/main.scss',
     },
   },
+  min: {
+    options: {
+      outputStyle: 'compressed'
+    },
+    files: {
+      'public/dist/main.min.css': 'public/scss/main.scss',
```

Comment:
Yep later, although this one is less of a performance hit compared to uglify. (which is why we don't want to have to wait for the uglification on watching for example).